ramls/traits/filterable.raml

```diff
@@ -9,8 +9,8 @@ queryParameters:
     displayName: Content type
     type: string
     description: |
-      Filter to narrow down results based on content type.
-      Possible values are all, aggregatefulltext, abstractandindex, ebook, ejournal, print, unknown, onlinereference.
+      Filter to narrow down results based on content type. Defaults to all.
+      Possible values are all, aggregatedfulltext, abstractandindex, ebook, ejournal, print, unknown, onlinereference.
     example: ebook
```

Comment:
Good catch on the `aggregatedfulltext` -- wonder if it is possible to define these as an enum (similar to how publicationTypes are handled in titles)

Reply:
great idea, will do